blocks/card-list-block/themes/news.json

```diff
@@ -39,7 +45,10 @@
 	"card-list-list": {
 		"styles": {
 			"default": {
-				"padding": "0 var(--global-spacing-4)"
+				"padding-block-end": "0",
+				"padding-block-start": "0)",
```

Comment:
There's an extra parenthesis in the value that's breaking the SCSS compilation.

Reply:
Updated [985f546](https://github.com/WPMedia/arc-themes-blocks/pull/1830/commits/985f5462233dc51d4d644a3ce27cb5feedc04e40)